### PR TITLE
Remove pip-tools and re-compile requirements

### PR DESCRIPTION
**Pull request by fievelk**
Adding `pip-tools` to the requirements is considered to be unsafe and it's generating errors/warnings because of incompatibilities between some of our dependencies. Updating `pip-tools` should be manually done by the developer before installing the requirements.

- remove `pip-tools` from explicit requirements
- re-compile requirements (using `fab compile`)

### CODE REVIEW

File: requirements/pandapower.txt [hunk 1]
```diff
@@ -1,79 +1,263 @@
...
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   tox
+pre-commit==2.14.0
+    # via -r requirements/dev.txt
+psutil==5.8.0
+    # via -r requirements/dev.txt
+py==1.9.0
+    # via
```
Comment (hannesdiedrich) [inline requirements/pandapower.txt:146]: I really wonder who came up with this default setting to distribute `via -r requirements/dev.txt cffi` into new lines. Must be somebody who is paid by the added number of lines.

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: requirements/pandapower.txt (hunk 1 above)
Comment (fievelk) [reply requirements/pandapower.txt:146]: It was done to avoid git conflicts, since they are more likely to happen in long lines :D

**Review by faizan2590 — APPROVED** [review]
LGTM

**Pull request merged**